src/intents.cs

```diff
@@ -6022,23 +6086,19 @@ interface INSearchForPhotosIntentResponse {
 		NSNumber SearchResultsCount { get; set; }
 	}
 
-#if XAMCORE_4_0
-	[NoMac]
-#elif MONOMAC
-	[Obsoleted (PlatformName.MacOSX, 10,0, message: "Unavailable on macOS, will be removed in the future.")]
-#endif
 	[iOS (10, 0)]
```

Comment:
Shouldn't this type also get a `[Mac (12,0)]`?

Reply:
yes, but there are some deprecated types that had earlier availability and deprecation in xamarin.ios. I wasn't sure if I could just delete those entirely, but assumed it might be a breaking change and that it would be better to just decorate everything else with Mac (12,0). Is there an alternative, better solution though?

Reply:
I'm not sure I understand. In all the other cases where you removed a similar block of availability attributes for macOS ([NoMac] in XAMCORE_4_0 + [Obsolete] in MONOMAC), you added a `[Mac (12, 0)]`. That's correct. However, this particular type didn't get a `[Mac (12, 0)]` - but you still removed the previous availability attributes. That effectively means that you're saying that this type was introduced in macOS in the first macOS version we support (macOS 10.9), and I don't think that's what you wanted :smile:

Reply:
ahh ok I misunderstood, but I see what you mean now!